### ISSUE TITLE
"space-in-brackets" does not catch import / export brackets

### ISSUE THREAD

**Issue by bytasv** [eslint/eslint#2216]
When using "space-in-brackets" options following code does not throw neither error, nor warning:
```
import {something} from '../somewhere';
// ....
export {something}
```

**Comment by nzakas** [comment]
Looks like this rule didn't get fixed for ES6 yet.

**Comment by nzakas** [comment]
Working on this.
